tutorials/32_Classifying_Documents_and_Queries_by_Language.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>It seems like the <code>DocumentLanguageClassifier</code> wasnt able to detect the French docs. Can you check what's wrong?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>